powerai_verification_tool/powerai_verification_tool.sh

```diff
@@ -0,0 +1,790 @@
+	  echo " Anaconda is installed sucessfully."  2>&1 >> "${REPFILE}"
+	  echo " Anaconda version: ${anaconda_ver} configured sucessfully." 2>&1 >> "${REPFILE}"
+      else
+          echo " Anaconda version: ${anaconda_ver} is supported for POWERAI $powerai_version but Anaconda ${anaconda_version1} is configured" 2>&1 | tee -a "${LOGFILE}"
+     fi
+
+}
+
+validate_anaconda
+
+
+IS_MLDL_REPO_INSTALLED="NO"
+echo " Check for mldl-repo-local installation." 2>&1 >> "${LOGFILE}"
+sudo rpm -qa | egrep 'mldl-repo' > /dev/null
+if [ $? -eq 0 ]
+then
+   echo " mldl-repo-local installed sucessfully" 2>&1 >> "${LOGFILE}"
+   IS_MLDL_REPO_INSTALLED="YES"
+else
+   echo " mldl-repo-local is not installed " 2>&1 | tee -a "${LOGFILE}"
+   IS_MLDL_REPO_INSTALLED="NO"
+   exit 1
+fi
+
+POWER_MLDL_REPO_INSTALLED="NO"
+echo " Check for power-mldl installation." 2>&1 >> "${LOGFILE}"
+sudo rpm -qa | egrep 'power-mldl' > /dev/null
+if [ $? -eq 0 ]
+then
+   echo " power-mldl installed sucessfully" 2>&1 >> "${LOGFILE}"
+   POWER_MLDL_REPO_INSTALLED="YES"
+else
+   echo " power-mldl is not installed " 2>&1 | tee -a "${LOGFILE}"
+   POWER_MLDL_REPO_INSTALLED="NO"
+   exit 1
+fi
+
+if [ -e $LICENSE_BIN/check-powerai-license.sh ]; then
+  $LICENSE_BIN/check-powerai-license.sh
+  if [ $? -ne 0 ]
+  then
+      echo " PowerAI license is to be accpeted to activate the frameworks." 2>&1 | tee -a  "${LOGFILE}"
+      POWERAI_LICENSE_ACCEPTED="NO"
+      exit 1
+  else
+     echo " PowerAI license is accpeted." 2>&1 >> "${LOGFILE}"
+     echo " PowerAI installed sucessfully" 2>&1 | tee -a "${REPFILE}"
+     POWERAI_LICENSE_ACCEPTED="YES"
+  fi
+else
+   echo " PowerAI is not installed properly." 2>&1 | tee -a  "${LOGFILE}"
+   exit 1
+fi
+
+
+echo " Is PowerAI License accepted: ${POWERAI_LICENSE_ACCEPTED}" 2>&1 >> "${REPFILE}"
+
+
+
+
+validate_ddl(){  
+    # Initialize report vars
+    #IS_DDL_INSTALLED="0"
+    IS_DDL_INSTALLED_REP="NO"
+    DDL_INFO=""
+    IS_DDL_ACTIVE="0"
+    IS_DDL_ACTIVE_REP="NO"
+
+    IS_DDL_INSTALLED=`sudo rpm -qa | grep power-ddl | wc -l`
+    if [ "${IS_DDL_INSTALLED}" != "0" ]
+    then
+        echo " DDL is installed." 2>&1 >> "${LOGFILE}"
+        DDL_INFO=`sudo yum list | grep -E '(^|\s)power-ddl.ppc64le($|\s)'`
+        IS_DDL_INSTALLED_REP="YES"
+    else
+       echo " DDL is not installled."  2>&1 | tee -a "${LOGFILE}"
+       #exit 1
+   fi
+   
+   # If tensoflow is installed, on PowerAI we need to activate it
+   if [ "${IS_DDL_INSTALLED}" != "0" ]
+   then
+       if [ $POWERAI_LICENSE_ACCEPTED = "YES" ]
+       then
+           echo " Activating DDL." 2>&1 >> "${LOGFILE}"
+           source /opt/DL/ddl/bin/ddl-activate
+           if [ $? -eq 0 ]
+           then
+             	 echo " DDL activated sucessfully" 2>&1 >> "${LOGFILE}"
+          	 IS_DDL_INSTALLED_REP="YES"
+       	   else
+          	 /opt/DL/tensorflow/bin/install_dependencies -y
```

Comment:
We shouldn't be installing dependencies here.

Reply:
@pvemuri1 I suggest you to create our own conda environment pai_verify_env within this tool after validate_anaconda() is called. Then any framework verification/validation can actually perform install_dependencies with in this conda environment pai_verify_env and we can clean-up/remove this conda environment pai_verify_env as the end of execution of this verification tool.